### ISSUE TITLE
✔️ 🐞 Bug Report: Dynamic Post Image Attachments on Gallery and Image Carousel widgets not working after updating to v3.11

### ISSUE THREAD

**Issue by PaulSchroeder** [elementor/elementor#21314]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

After I update to v3.11, galeries and sliders using dynamic content "media attached to post" don't catch any media data.

### Steps to reproduce

1/ Creata a page
2/ upload some images
3/ Add a gallery widget
4/ choose dynamic content : media attached to post

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.20.2
	MySQL version: MariaDB Server v10.3.35
	PHP Version: 7.3.33
	PHP Memory Limit: 128M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://www.hlg.lu
	Home URL: https://www.hlg.lu
	WP Multisite: No
	Max Upload Size: 2 Mo
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: fr-FR
	Timezone: Europe/Luxembourg
	Admin Email: hello@youtag.lu
	Debug Mode: Inactive

== Theme ==
	Name: HLG
	Version: 1.0
	Author: Youtag
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.6.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: fr_FR
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.0 Safari/605.1.15

== Active Plugins ==
	Category Order and Taxonomy Terms Order
		Version: 1.7.4
		Author: Nsp-Code

	Elementor
		Version: 3.11.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.10.1
		Author: Elementor.com

	HumansNotBots - Easy, Accessible Email Cloaker
		Version: 3.2
		Author: Sophiah (Zing-Ming)

	IP Vault – WP Firewall
		Version: 1.1
		Author: Paul C. Schroeder

	Relevanssi
		Version: 4.18.3
		Author: Mikko Saari

	WP Super Cache
		Version: 1.9.3
		Author: Automattic

	Yoast SEO
		Version: 20.1
		Author: Team Yoast


== Utilisation des éléments ==
	
	archive : 1
		container : 4
		heading : 2
		portfolio : 1
		shortcode : 1
	container : 6
		button : 4
		container : 9
		heading : 1
		loop-grid : 3
		portfolio : 1
		shortcode : 1
		theme-post-content : 1
		theme-post-title : 1
	footer : 1
		container : 1
		icon-list : 1
		nav-menu : 1
		text-editor : 1
		theme-site-logo : 1
	header : 1
		container : 1
		nav-menu : 1
		search-form : 1
		theme-site-logo : 1
	widget : 0
		loop-grid : 1
	wp-page : 4
		container : 31
		divider : 2
		google_maps : 1
		heading : 9
		image : 5
		portfolio : 1
		shortcode : 1
		spacer : 2
		testimonial-carousel : 1
		text-editor : 7
		theme-post-title : 1
	single-post : 1
		container : 3
		image-carousel : 1
		post-info : 1
		post-navigation : 1
		spacer : 2
		theme-post-content : 1
		theme-post-title : 1
	single-page : 1
		container : 1
		theme-post-content : 1
	loop-item : 1
		container : 3
		image : 1
		spacer : 1
		theme-post-content : 1
		theme-post-title : 1


== Réglages ==
	
	cpt_support: post, page
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	font_display: swap


== Fonctionnalités ==
	Custom Fonts: 0
	Custom Icons: 0

== Intégrations ==
	


== Expériences d’Elementor ==
	Sortie DOM optimisée: Actif par défaut
	Amélioration du chargement des ressources: Actif par défaut
	Optimisation du chargement du CSS: Actif par défaut
	Icônes de police « inline »: Actif
	Améliorations de l’accessibilité: Actif par défaut
	Points d’arrêt personnalisés supplémentaires: Actif par défaut
	admin_menu_rearrangement: Inactif par défaut
	Conteneur Flexbox: Actif
	Upgrade Swiper Library: Inactif par défaut
	Par défaut vers le Nouveau constructeur de thème: Actif par défaut
	En-tête et pied de page du thème Hello: Actif par défaut
	Editor V2: Inactif par défaut
	Page d’atterrissage: Actif par défaut
	Éléments imbriqués: Actif
	Lazy Load Background Images: Inactif par défaut
	Définir comme valeur par défaut: Actif par défaut
	Transitions des pages: Actif par défaut
	Notes: Actif par défaut
	Boucle: Actif par défaut
	Loop Carousel: Inactif par défaut
	Envois de formulaire: Actif par défaut
	Accroche de défilement: Actif par défaut


== Journal ==
	
PHP: showing 10 of 10PHP: 2023-01-30 18:10:27 [notice X 56][/var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/core/common/modules/ajax/module.php::172] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-30 20:13:03 [notice X 45][/var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined index: condition_type [array (
  'trace' => '
#0: /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-01-30 20:13:04 [notice X 34][/var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::84] Undefined index: elementor-preview [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-31 10:20:28 [error X 1][/var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/includes/managers/widgets.php::425] Uncaught Error: Call to a member function is_editable_by_current_user() on bool in /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/includes/managers/widgets.php:425
Stack trace:
#0 [internal function]: Elementor\Widgets_Manager->ajax_render_widget(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#1 /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/core/common/modules/ajax/module.php(172): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /var/www/vhosts/hlg.lu/httpdocs/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#3 /var/www/vhosts/hlg.lu/httpdocs/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#4 /var/www/vhosts/hlg.lu/httpdocs/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /var/www/vhosts/hlg.lu/httpdocs/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-31 13:32:42 [error X 1][/var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on bool in /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:295
Stack trace:
#0 /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/core/common/modules/ajax/module.php(172): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /var/www/vhosts/hlg.lu/httpdocs/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /var/www/vhosts/hlg.lu/httpdocs/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Ar [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-31 17:02:34 [notice X 3][/var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor-pro/modules/query-control/classes/elementor-post-query.php::247] Undefined property: WP_Error::$taxonomy [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-01 12:25:02 [notice X 475][/var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor-pro/modules/posts/skins/skin-content-base.php::264] Trying to get property 'ID' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-08 18:01:08 [error X 1][/var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/core/wp-api.php::26] Uncaught Error: Call to undefined function Elementor\Core\get_plugins() in /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/core/wp-api.php:26
Stack trace:
#0 /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/modules/compatibility-tag/base-module.php(96): Elementor\Core\Wp_Api->get_plugins()
#1 /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/modules/compatibility-tag/base-module.php(58): Elementor\Modules\CompatibilityTag\Base_Module->get_plugins_with_header()
#2 /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/modules/compatibility-tag/module.php(57): Elementor\Modules\CompatibilityTag\Base_Module->get_plugins_to_check()
#3 /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/modules/compatibility-tag/base-module.php(148): Elementor\Modules\CompatibilityTag\Module->get_plugins_to_check()
#4 /var/www/vhosts/hlg.lu/httpdocs/wp-includes/class-wp-hook.php(308): Elementor\Modules\CompatibilityTag\Base_Module->Elementor\Modules\CompatibilityTag\{closure}('')
 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-13 16:23:06 [notice X 3][/var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/core/upgrade/elementor-3-re-migrate-globals.php::102] Undefined variable: active_kit_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-13 16:25:19 [error X 1][/var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/core/dynamic-tags/manager.php::444] Uncaught Error: Call to a member function get_content() on null in /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/core/dynamic-tags/manager.php:444
Stack trace:
#0 [internal function]: Elementor\Core\DynamicTags\Manager->ajax_render_tags(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#1 /var/www/vhosts/hlg.lu/httpdocs/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /var/www/vhosts/hlg.lu/httpdocs/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#3 /var/www/vhosts/hlg.lu/httpdocs/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#4 /var/www/vhosts/hlg.lu/httpdocs/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /var/www/vhosts/hlg.lu/httpdocs/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 4 of 4JS: 2023-01-31 10:38:29 [error X 6][https://www.hlg.lu/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.10.2:3:666866] undefined is not an object (evaluating 'C.getControlModel(T).get') 
JS: 2023-01-31 11:35:16 [error X 1][https://www.hlg.lu/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.10.2:3:296806] undefined is not an object (evaluating 'T.params.args.data.id') 
JS: 2023-01-31 12:54:45 [error X 51][https://www.hlg.lu/wp-admin/load-scripts.php?c=0&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:4:51823] undefined is not an object (evaluating 'a.replace') 
JS: 2023-01-31 16:12:27 [error X 6][https://www.hlg.lu/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.10.2:3:782338] undefined is not an object (evaluating 'this.store.find((function(T){return T.original===C})).localized') 

Log: showing 18 of 182023-02-13 15:07:24 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.2',
  'to' => '3.11.0',
)]
2023-02-13 15:08:27 [info] elementor::elementor_updater Started 
2023-02-13 15:08:27 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-13 15:08:28 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-13 15:08:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.2',
  'to' => '3.11.0',
)]
2023-02-13 15:27:22 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.3',
  'to' => '3.11.0',
)]
2023-02-13 15:28:29 [info] elementor-pro::elementor_pro_updater Started 
2023-02-13 15:28:29 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-02-13 15:28:29 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-13 15:28:29 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.3',
  'to' => '3.11.0',
)]
2023-02-13 16:23:06 [notice] Re-migrate 3.0.0 globals: Start 
2023-02-13 16:23:06 [notice] Re-migrate 3.0.0 globals: _v_3_0_0_move_default_typography_to_kit 
2023-02-13 16:23:06 [notice] System typography is disabled. nothing to do. 
2023-02-13 16:23:06 [notice] Re-migrate 3.0.0 globals: _v_3_0_0_move_default_colors_to_kit 
2023-02-13 16:23:06 [notice] System colors are disabled. nothing to do. 
2023-02-13 16:23:06 [notice] Re-migrate 3.0.0 globals: _v_3_0_0_move_saved_colors_to_kit 
2023-02-13 16:23:06 [notice] Custom colors already exist. nothing to do. 
2023-02-13 16:23:06 [notice] Re-migrate 3.0.0 globals: End 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibilité non spécifiée

== Elementor Pro - Compatibility Tag ==
	

				
````

</details>

**Comment by nicholaszein** [comment]
Hello @PaulSchroeder! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by nicholaszein** [comment]
Hi @PaulSchroeder!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.11.1**! 🥳 

✔️ PFeel free to check it out and update your plugin to the new version!

Cheers 🥂
